static/js/stream_popover.js

```diff
@@ -500,6 +533,27 @@ exports.register_topic_handlers = function () {
 
         e.stopPropagation();
     });
+
+    $('body').on('click', '.sidebar-popover-move-topic-messages', function (e) {
+        const topic_row = $(e.currentTarget);
+        const stream_id = topic_row.data('stream-id');
+        const topic_name = topic_row.data('topic-name');
+        const message = home_msg_list.all_messages()
+            .find(m => m.topic === topic_name && m.stream_id === stream_id);
+        build_move_topic_to_stream_popover(e, message.id, stream_id);
+        e.stopPropagation();
+    });
+
+    $('body').on('click', '.popover-move-topic-to-stream', function (e) {
+        const stream_row = $(e.currentTarget);
+        const stream_id = stream_row.data('stream-id');
+        const message_id = stream_row.data('message-id');
+        message_edit.show_topic_edit_spinner(stream_row);
```

Comment:
Have you tested this spinner behavior looks right (e.g. by adding a `sleep(5)` in the backend endpoint)?

Reply:
It doesn't work. Let me double check it!